### ISSUE TITLE
Bug: Autocomplete, Version: 5.2.3

### ISSUE THREAD

**Issue by Lantok** [dbeaver/dbeaver#4509]
When you select an entry from auto-completion suggestions, the selected item is placed before the text that generated these suggestions instead of replacing it.

**Comment by benborba** [comment]
same here

**Comment by lemon-juice** [comment]
#4482

**Comment by novusnick** [comment]
Seeing the same issue.  Also, trying to select the table name from autocomplete doesn't work (oddly, selecting the database does though)

**Comment by zakaluka** [comment]
Even worse, when double-clicking a suggestion, the text is inserted by deleting the following "symbol".  For example,

```sql
select * from dbo.<autocomplete showing Table1>;

select * from dbo.Table2;
```

becomes

```sql
select * from dbo.Table1select * from dbo.Table2;
```

**Comment by serge-rider** [comment]
Already fixed (see #4482)
You can download 5.2.3 again (latest build contains the fix)
